packages/client/src/theme.ts

```diff
@@ -1,17 +1,60 @@
 import { LinkProps } from '@mui/material';
+import { green, lime } from '@mui/material/colors';
 
 import { LinkBehavior } from './linkBehavior';
 
 type PaletteMode = 'light' | 'dark';
 
 const themeColors = {
   primary: {
-    main: '#1b5e20',
+    main: green[900],
+    background: 'rgba(27,94,32,8%)',
   },
   secondary: {
-    main: '#cddc39',
+    main: lime[500],
   },
 };
+const commonComponents = {
+  MuiLink: {
+    defaultProps: {
+      component: LinkBehavior,
+    } as LinkProps,
+  },
+  MuiButtonBase: {
+    defaultProps: {
+      LinkComponent: LinkBehavior,
+    },
+  },
+  MuiButton: {
+    styleOverrides: {
+      root: {
+        height: '48px',
```

Comment:
To rozwaliło button na stronie głównej (Wyszukaj). Samą zmianę popieram bo zmniejszy powtarzalność ale popraw go proszę.

Reply:
@tchojnacki Poprawiłam i zmieniłam jeszcze jedną rzecz w komponencie `Hero` czyli położenie tego buttonu do scrolla do następnej sekcji i zawinęłam tekst. Poza scope, ale tak mi się wydaje, że teraz lepiej wygląda.